### PR TITLE
Try osx-arm64 migration again.

### PR DESCRIPTION
**Pull request by erykoff**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,16 +7,19 @@ package:
...
+  number: 1
   skip: true  # [win]
 
 outputs:
   - name: rubin_sim
     version: {{ version }}
     requirements:
+      build:
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
```
Comment (beckermr) [inline recipe/meta.yaml:22]:
```suggestion
```

**Review by beckermr — COMMENTED** [review]
Trying this.

**Comment by erykoff** [comment]
woo-hoo!

**Pull request merged**